include/class_info.php

```diff
@@ -269,8 +271,6 @@ public static function reversalsMissing($arrIndexed, $arrReversalsImported)
 		{
 			$status .= "<div style=\"clear:both;\"><div style=\"text-align:right; float:left;\"><nobr>" . $indexed->language_code . ":</nobr></div><div style=\"float:left;\">&nbsp;". $indexed->totalIndexed;
 
-			array_filter($arrReversalsImported, function($el) { return $el->post_id == 0; });
```

Comment:
Removed this line because it doesn't do anything. The `array_filter` function doesn't modify the original array, and the return value isn't used.